api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -98,6 +98,14 @@ void shouldProcessClaimFromPregnancyToChildTurningFour() throws JsonProcessingEx
         claim = repositoryMediator.loadClaim(claimId);
         assertThat(claim.getClaimStatus()).isEqualTo(ClaimStatus.EXPIRED);
         assertThat(claim.getClaimStatusTimestamp()).isAfter(LocalDateTime.now().minusHours(1));
+        assertThat(claim.getCardStatus()).isEqualTo(CardStatus.PENDING_CANCELLATION);
+        assertThat(claim.getCardStatusTimestamp()).isAfter(LocalDateTime.now().minusHours(1));
```

Comment:
What do these checks achieve over and above just doing `assertThat(claim.getCardStatusTimestamp()).isBefore(LocalDateTime.now());`? I know its the same as the above timestamp check, but I can't remember why that one is like that either.

Reply:
Updated to assert against a local variable assigned to LocalDateTime.now()